openedx/core/djangoapps/content/course_overviews/migrations/0010_auto_20160329_2317.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('course_overviews', '0009_readd_facebook_url'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='courseoverview',
+            name='facebook_url',
+        ),
```

Comment:
I'm a bit confused about this.  The original PR did have migration code for it.  However, since then, the devOps team made a change to revert it in order to support production deployment to multiple workers, concurrently running different versions of the code.  @jibsheet At this point, can we go ahead and re-add the removal of this field in the migration file?